### PR TITLE
Godot linter

### LINKED ISSUE #2268: Make the `godot` linter rule more restrictive.
### Description

Ref: #1541 

When [enabling](https://github.com/percona/pmm/pull/2252) the `godot` rule, we have intentionally chosen to set its `period` config [parameter](https://github.com/percona/pmm/pull/2252/files#diff-6179837f7df53a6f05c522b6b7bb566d484d5465d9894fb04910dd08bb40dcc9R32) to `false`, since otherwise the PR would be too heavy to review.

We want to turn it back to `true`, i.e. `period: true` so as to make the linter fail when it encounters the absence of the dot at the end of a comment.

This seems to follow Go's conventions and best practices.

### Suggested solution

1. set the `period: true` in the `.golangci.yml` file
2. fix the linter warnings

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by vishwas-sharma2480**
Description:
Ref: closes #2268 

Summary:
When enabling the Godot rule, the period config parameter was intentionally set to false to avoid making the pull request too heavy for review. This change aims to turn it back to true (period: true) in the .golangci.yml file. This adjustment will cause the linter to fail when it encounters the absence of a period at the end of a comment, aligning with Go's conventions and best practices.

Suggested Solution:

Set the period parameter to true in the .golangci.yml configuration file.
Address and resolve any resulting linter warnings.
Additional Context:
I have already fixed the issue and am prepared to create this pull request. The proposed changes align with Go's coding conventions and should enhance the quality of the codebase.

### CODE REVIEW

File: .github/workflows/codesee-arch-diagram.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# This workflow was added by CodeSee. Learn more at https://codesee.io/
```
Comment (artemgavrilov) [inline .github/workflows/codesee-arch-diagram.yml:1]: Please remove this workflow. Any CI changes should be proposed in separate PR. But I'm not sure that we ready to add something like this now.

Comment (atymchuk) [inline .github/workflows/codesee-arch-diagram.yml:1]: I don't think we can use this action for a number of reasons, one of which being vendor validation, which is an entire process, that entails a request, a justification, a security check, and more.

Can we skip this for now?

File: agent/runner/actions/common.go [hunk 2]
```diff
@@ -40,7 +40,7 @@ var whiteSpacesRegExp = regexp.MustCompile(`\s+`)
 //	["value 1", 2, …]
 //	…
 //
-// ]
+// ].
```
Comment (artemgavrilov) [inline agent/runner/actions/common.go:43]: That one is weird, but ok 😁

**Comment by atymchuk** [comment]
> Description: Ref: #1541

It refers rather to #2268

**Review by artemgavrilov — CHANGES_REQUESTED** [review]
Great job! 👏

Can you please address my comment about CI and resolve conflicts? Thank you!

**Comment by vishwas-sharma2480** [comment]
HI @artemgavrilov I have removed the file from my repo should i raise a new request

**Comment by artemgavrilov** [comment]
> HI @artemgavrilov I have removed the file from my repo should i raise a new request

Great, now only conflicts resolving is left. Once its done just click `Re-request review` for me and Alex.

**Comment by artemgavrilov** [comment]
@vishwas-sharma2480 Conflicts should be resolved before merging. In this case you need to merge(sync) upstream to your fork and resolve conflicts there.

**Comment by atymchuk** [comment]
I was able to resolve the conflicts just by merging the `main` branch to this one. Could you please try the same?

**Comment by vishwas-sharma2480** [comment]
Hi Artem

But for that I might need access to merge to main
Best regards
Vishwas

**Comment by vishwas-sharma2480** [comment]
Hi @atymchuk 
I merged from my end and  i am not seeing any conflicts now

**Review by artemgavrilov — APPROVED** [review]

**Review by artemgavrilov — APPROVED** [review]
CI failure is unrelated to changes in this PR. 
We can merge it.

**Review by atymchuk — APPROVED** [review]

**Pull request merged**